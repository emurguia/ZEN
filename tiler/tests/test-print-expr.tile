// This tests that print is able to handle expressions

init {
	int x;
	int y;
	float a;
	float b;

	tile(3,3);
	x = 1;
	y = 1;
	a = 3.0;
	b = 4.0;
	iprint(x*y/y-y);
	fprint(b/a);
	close();
}