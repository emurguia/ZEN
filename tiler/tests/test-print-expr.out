0
1.333333
The game window was successfully launched ...
The game window has been exited. Program exited.
Game over.
